dockerfiles/cli/scripts/cli.sh

```diff
@@ -111,11 +111,52 @@ cmd_init_reinit_pre_action() {
   fi
 }
 
+check_all_ports(){
+  docker run -d -p 80:80 \
+                -p 443:443 \
+                -p 2181:2181 \
+                -p 5000:5000 \
+                -p 23750:23750 \
+                -p 23751:23751 \
+                -p 32001:32001 \
+                -p 32101:32101 \
+                --name fake ${UTILITY_IMAGE_ALPINE} \
+                                 httpd -f -p 80 \
+                                          -p 443 \
```

Comment:
That is what you need to run if you are native on the host.  However, the command that is in this PR is designed for running inside of a container and then testing the host's bound ports.  Just did a test on Mac and it worked as expected.